api/src/Security/Voter/CampRoleVoter.php

```diff
@@ -26,11 +29,12 @@ class CampRoleVoter extends Voter {
 
     public function __construct(
         private EntityManagerInterface $em,
+        private ResponseTagger $responseTagger
     ) {}
 
     protected function supports($attribute, $subject): bool {
         return in_array($attribute, array_keys(self::RULE_MAPPING))
-            && ($subject instanceof BelongsToCampInterface || $subject instanceof BelongsToContentNodeTreeInterface);
+            && ($subject instanceof BelongsToCampInterface || $subject instanceof BelongsToContentNodeTreeInterface || $subject instanceof Request);
```

Comment:
Why do we vote on the requests themselves now too?

Reply:
Because of this line: https://github.com/ecamp/ecamp3/pull/3610/files#diff-b8aa5b1f09a6c2988c4df97965833e4bf7b2f89106e4ff784c60ee7a8365f45cR59

This enables us to return a 403 if I access /camp/123/categories but I don't a have access to /camp/123.

A nicer implementation would be possible with https://github.com/api-platform/core/pull/5290

Reply:
https://github.com/api-platform/core/pull/5290 has been merged, so this is now not needed anymore. Reverted in commit https://github.com/ecamp/ecamp3/pull/3610/commits/7703e00507cb6b440cd063e2c9d84946948f21ef